cmd/presets/configs.go

```diff
@@ -300,11 +300,13 @@ commands:
 questions:
   compose:
     - key: app
-      default_answer: PHP 7.4
+      default_answer: PHP 8.0
       message: What app service do you want to use
```

Comment:
@fabriciojs - this does not appear to be the latest parsing of the presets, because the `message` should have been changed to `message: What PHP version do you want to use`.